pkg/api/validation.go

```diff
@@ -0,0 +1,5 @@
+package api
+
+func ValidateRange(period int32) bool {
```

Comment:
Since this is a boolean function, can we rename this `IsValidRange`?

That typically reads better:

```go
if !api.IsValidRange(opts.Period)
```

Reply:
thanks for the feedback!